inginious/frontend/templates/task.html

```diff
@@ -436,6 +439,41 @@ <h4 class="modal-title" id="socialModalLabel">$:_("Share my result on: ")</h4>
 
 </script>
 
+<script type="text/javascript">
+    $$("#load_loc").on('click', function(e){
+        alert("Loaded from browser");
```

Comment:
Is this supposed to actually display the message box ?

Reply:
Removed... was for test purposes